lab_4_summarization_textrank/main.py

```diff
@@ -313,4 +470,11 @@ def reply(self, query: str, n_summaries: int = 3) -> str:
         :param n_summaries: the number of summaries to include in the answer
         :return: the answer
         """
-        pass
+        if not query or not isinstance(query, str):
+            raise IncorrectQueryError('Incorrect query. Use string as input.')
+        check_type(n_summaries, int)
+        if n_summaries > len(self._knowledge_database):
+            raise ValueError
+        keywords = self._sentence_preprocessor.preprocess_text(query)
+        close_texts = self._find_texts_close_to_keywords(keywords, n_summaries)
+        return 'Ответ:\n' + '\n\n'.join(self._knowledge_database[text]['summary'] for text in close_texts)
```

Comment:
use fstring

Reply:
i'm not if this is possible to be done, as using \n inside the {brackets} leads to an error